pennylane/utils.py

```diff
@@ -433,3 +433,22 @@ def expand_vector(vector, original_wires, expanded_wires):
     expanded_tensor = np.moveaxis(expanded_tensor, original_indices, wire_indices)
 
     return expanded_tensor.reshape(2 ** M)
+
+
+def _hash_iterable(iterable):
+    """Returns a single hash of an input iterable.
+
+    The iterable must be flat and can contain only numbers and NumPy arrays.
+
+    Args:
+        iterable (Iterable): the iterable to generate a hash for
+
+    Returns:
+        int: the resulting hash
+    """
+    hashes = []
+    for obj in iterable:
+        to_hash = (obj.tobytes(), obj.shape) if isinstance(obj, np.ndarray) else obj
+        obj_hash = hash(to_hash)
+        hashes.append(obj_hash)
+    return hash(tuple(hashes))
```

Comment:
I don't know why codecov is saying this, since the tests in `tests/test_utils.py` should be covering this - indeed my local coverage says that these lines are covered.

Reply:
@trbromley Codecov has been doing the same to me 🤔 I've been posting screenshots of my coverage in the PRs to allay everyone's fears, it's making me paranoid!